Godeps/_workspace/src/github.com/docker/docker/daemon/graphdriver/zfs/zfs_linux.go

```diff
@@ -15,7 +15,7 @@ func checkRootdirFs(rootdir string) error {
 	}
 
 	if graphdriver.FsMagic(buf.Type) != graphdriver.FsMagicZfs {
-		logrus.Debugf("[zfs] no zfs dataset found for rootdir '%s'", rootdir)
+		logrus.Debugf("[zfs] no zfs dataset found for rootdir '%s', found '%s'", rootdir, buf.Type)
```

Comment:
This code is vendored from upstream, so has to be changed here; https://github.com/moby/moby/blob/master/daemon/graphdriver/zfs/zfs_linux.go#L18